packages/components/src/templates/teaser.stories.ts

```diff
@@ -0,0 +1,335 @@
+        breakpoint="9999"
+        inset
+        class="flex-1 flex flex-col max-w-[426.67px]"
+      >
+        <div slot="media" class="relative">
+          <img
+            src="./placeholders/images/collaboration.jpg"
+            alt="Two men in business attire sitting on a sofa in a modern office, smiling and looking at a laptop screen."
+            class="aspect-video object-cover"
+          />
+        </div>
+        <h3 slot="headline" class="sd-headline sd-headline--size-3xl">Expert views</h3>
+        <div class="flex flex-col gap-5 mt-4">
+          <p>
+            Our experts assess current economic developments and topics for your investment decision. All analyses,
+            white papers and studies can be found here.
+          </p>
+          <div class="flex-none">
+            <sd-button href="javascript:void(0)" variant="primary">To expert views</sd-button>
+          </div>
+        </div>
+      </sd-teaser>
+    `;
+  }
+};
+
+/**
+ * If the teaser itself should be clickable and there are no other links inside, then wrap the teaser with an anchor tag.
+ * The button on the bottom of the teaser is **must not be a link**. According to the HTML spec, it is not allowed to
+ * have an anchor tag inside another anchor.
+ *
+ * ```html
+ * <a href="#">
+ *    <sd-teaser class="interactive">
+ *      <sd-button>CTA</sd-button>
+ *    </<sd-teaser>
+ * </a>
+ * ```
+ *
+ * To further highlight the interactive state of the teaser, the opacity of the media part can be reduced on hover.
+ *
+ * ```css
+ * sd-teaser.interactive:hover {
+ *   &::part(media) {
+ *     transition-property: opacity;
+ *     transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+ *     transition-duration: 150ms;
+ *     transition-duration: 300ms;
+ *     opacity: 0.5;
+ *   }
+ * }
+ * ```
+ */
+
+export const ClickableTeaserWithLink = {
+  name: 'Clickable Teaser with Link',
+  render: () => {
+    return html`
+      <style>
+        sd-teaser.interactive:hover {
+          &::part(media) {
+            opacity: 0.7;
+          }
+        }
+      </style>
+      <a href="#" target="_blank" class="flex-1">
+        <sd-teaser variant="primary-100" breakpoint="9999" inset class="interactive max-w-[426.67px]">
+          <div slot="media" class="relative">
+            <img
+              class="aspect-video object-cover"
+              src="./placeholders/images/family.jpg"
+              alt="A man with glasses and a beard holding two young children in his arms, smiling outdoors with trees in the background."
+            />
+          </div>
+          <h3 slot="headline" class="sd-headline sd-headline--size-lg">Invest sustainably</h3>
+          <div class="flex flex-col gap-5 mt-4">
+            <p>How we are actively shaping the transformation of the economy in the interests of our investors.</p>
+            <div class="flex-none">
+              <sd-link href="javascript:void(0)">Find out more</sd-link>
```

Comment:
this example shows a link but above we say it **must not be a link**... In addition, the link text lets me assume there is a link behind it...

Reply:
can we use an example of a shopping cart item? eg. the teaser itself links to the item and the button adds it to the shopping cart...

Reply:
Please see my comment above. If correct then the sample is good as it is.

Reply:
Karl instead of "adding to cart" I propose adding a button saying "Download Annual Report" I think it resonates better with the stock images that we have.